dgl_sparse/include/sparse/sparse_format.h

```diff
@@ -18,42 +18,71 @@
 namespace dgl {
 namespace sparse {
 
-/** @brief SparseFormat enumeration */
+/** @brief SparseFormat enumeration. */
 enum SparseFormat { kCOO, kCSR, kCSC };
 
-/** @brief CSR sparse structure */
+/** @brief COO sparse structure. */
+struct COO {
+  /** @brief the dense shape of the matrix. */
```

Comment:
* All comments should be a valid sentence, e.g., capitalize the first word.
* There is no difference between dense or sparse shape so just say "The shape of the matrix." is fine.